games-board/chessx/chessx-1.4.6.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A free and open source chess database application"
+DESCRIPTION="With ChessX you can operate your collection of chess games \
+in many ways: browse, edit, add, organize, analyze, etc."
+HOMEPAGE="https://chessx.sourceforge.net/"
+COPYRIGHT="1998-2005 Gilles Vollant
+	1999-2004 Shane Hudson
+	2005 Ejner Borgbjerg
+	2005-2007 Marius Roets
+	2005-2007 Heinz R. Hopfgartner
+	2005-2010 Michal Rudolf
+	2005-2011 Sergey A. Tachenov
+	2007 Rico Zenklusen
+	2008 Aliaksandr Antonik
+	2010 P. Sereno
+	2011 Nokia Corporation and/or it's subsidiary(-ies)
+	2012-2016 Jens Nissen"
```

Comment:
After some more time crafting script, here's the end result. Now if only someone could do the same for license...

- 2005-2016 ChessX developers
- 2014 Digia Plc and/or its subsidiary(-ies).
- 2005-2006 Ejner Borgbjerg
- 1998-2005 Gilles Vollant
- 2007 Heinz R. Hopfgartner
- 1990-2000 Info-ZIP
- 2012-2016 Jens Nissen
- 2005 Kamil Przybyla
- 2005-2007 Marius Roets
- 2014 Markus Kolb
- 2005-2010 Michal Rudolf
- 2011 Nokia Corporation and/or its subsidiary(-ies).
- 2006-2007 Pascal Georges
- 2010 P. Sereno
- 2007 Rico Zenklusen
- 2006-2007 Sean Estabrooks
- 2005-2011 Sergey A. Tachenov
- 1999-2004 Shane Hudson
- 2003 Sune Fischer
- 2006 Tobias Koenig
- 2005-2006 William Hoggarth